src/platforms/react-native/troubleshooting.mdx

```diff
@@ -168,3 +168,49 @@ If you're using `React Navigation 5` and you're not receiving transactions anymo
 ## RAM Bundles Mismatched Line Numbers
 
 If you experience mismatched line numbers on [sentry.io](https://sentry.io) when using RAM Bundles, this is due to a [bug](https://github.com/facebook/metro/issues/399) on the Metro tooling.
+
+## React Native 0.69 and higher
+
+If you're using React Native v0.69 or higher, the autolink has changed and we're working on a fix, follow the [issue](https://github.com/getsentry/sentry-react-native/issues/2267).
+
+### Android
```

Comment:
Is it only relevant for android? cus I see we changed ios config below as well

Reply:
It is for both, still a draft, figuring out iOS still.